src/als/mod.rs

```diff
@@ -1,15 +1,31 @@
 use itertools::Itertools;
 use std::collections::HashMap;
-use std::error::Error;
+
+use crate::ErrorBox;
 
 pub mod controller;
 pub mod iio;
 pub mod none;
 pub mod time;
 pub mod webcam;
 
-pub trait Als {
-    fn get(&self) -> Result<String, Box<dyn Error>>;
+#[allow(clippy::large_enum_variant)]
+pub enum Als {
+    Webcam(webcam::Als),
+    Iio(iio::Als),
+    Time(time::Als),
+    None(none::Als),
+}
```

Comment:
Just curious what you think about clippy's idea to Box the values here, do you think it's a good or bad idea, and correspondingly is there any significant downside to simply ignoring this warning?

Reply:
I don't think it matters much either way. As we only ever have one instance of each of these enums, we shouldn't be wasting any significant amount of memory. If we do box them, we do one more allocation on startup which also seems irrelevant. I decided for this to keep the code simple for now, but it's definitely up for discussion.

Reply:
I'm OK with the current implementation, just wanted to hear your opinion, thanks